homework04/homework04.py

```diff
@@ -0,0 +1,14 @@
+# Create a program that will tell you whether your input is a text or a number.
+# If it's a number the program must indicate whether it is even or odd.
+# If it's a text the program must specify its length.
+
+var = input('I will let you know whether you have typed a text or a number.\nGo ahead and type it now. ')
+
+if var.isdigit():
+    var = int(var)
+    if bool(int(var) % 2) is False:
+         print(f'{var} is a number. It\'s even.')
+    elif bool(int(var) % 2) is True:
```

Comment:
It is enough to use `if int(var) % 2:` as a condition here. 
And for binary situations, when a number can be even or odd (there is no third case) you can use `if ... else` statement